Update create-upload-generic.md to fix out-of-date information.

kelleymh opened the pull request:
Information near the top of this article incorrectly stated that all Linux images in the Azure image gallery (now the Azure Marketplace) are endorsed.  While this was true at one point, 3rd party partners have added other base Linux images to the Marketplace for non-endorsed distros.  This out-of-date information caused confusion among some Microsoft field people.

azurecla commented:
Hi __@magaal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mikegaal). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Jim-Parker commented:
@magaal thanks for the updates Mike.

Jim-Parker commented:
@SuperScottz hey Scott, could you please review this contribution?

SuperScottz commented:
@Jim-Parker @magaal Sorry for late, I think most of change in this PR has been updated, and I will double check first.

SuperScottz commented:
@magaal 
Mike
I have merged your PR update into this file, and created another PR https://github.com/Azure/azure-content-pr/pull/20445.
I think we could close this PR if you have no concern, thank you!

tysonn commented:
Closing based on @SuperScottz comment.

kelleymh committed:
Update create-upload-generic.md

Information near the top of this article incorrectly stated that all Linux images in the Azure image gallery (now the Azure Marketplace) are endorsed.  While this was true at one point, 3rd party partners have added other base Linux images to the Marketplace for non-endorsed distros.  This out-of-date information caused confusion among some Microsoft field people.
